packages/video_player/video_player_avfoundation/pigeons/messages.dart

```diff
@@ -23,12 +23,7 @@ class CreationOptions {
   String? uri;
   String? packageName;
   String? formatHint;
-  Map<String?, String?> httpHeaders;
-}
-
-class MixWithOthersMessage {
-  MixWithOthersMessage(this.mixWithOthers);
-  bool mixWithOthers;
```

Comment:
Was this just unused before?

Reply:
Yes, I somehow missed it in https://github.com/flutter/packages/pull/6507